src/Widget/Label.php

```diff
@@ -4,39 +4,78 @@
 
 namespace Yiisoft\Form\Widget;
 
-use Yiisoft\Arrays\ArrayHelper;
+use Yiisoft\Form\FormModelInterface;
 use Yiisoft\Form\Helper\HtmlForm;
-use Yiisoft\Form\Widget\Attribute\ModelAttributes;
 use Yiisoft\Html\Tag\Label as LabelTag;
 use Yiisoft\Widget\Widget;
 
 /**
  * Generates a label tag for the given form attribute.
  *
  * @link https://www.w3.org/TR/2012/WD-html-markup-20120329/label.html
+ *
+ * @psalm-suppress MissingConstructor
  */
 final class Label extends Widget
 {
-    use ModelAttributes;
+    private array $attributes = [];
```

Comment:
```suggestion
    private array $tagAttributes = [];
```

Reply:
https://github.com/terabytesoftw/yii-form/blob/883d98d085c527eb8387f1f2db126029ba0d4c0b/src/Widget/Label.php#L39 those are the global attributes of the widget.

Reply:
done.